keras_nlp/models/bert_test.py

```diff
@@ -45,6 +46,22 @@ def setUp(self):
             ),
         }
 
+        self.ds = tf.data.Dataset.from_tensor_slices(
```

Comment:
You should be able to `from_tensor_slices` directly from a dict I think? We could probably make this more concise.

Would `self.input_dataset = tf.data.Dataset.from_tensor_slices(input_data).repeat(5)` work?

Reply:
Also, let's maybe clean up the naming of these two input variables for clarity? `self.input_batch` for the one and `self.input_dataset` for this one maybe?

Reply:
Ah, yes. It works on dicts. Did not know that.

```input_batch = {
    "token_ids": tf.ones(
        (8, 512), dtype="int32"
    ),
    "segment_ids": tf.ones(
        (8, 512), dtype="int32"
    ),
    "padding_mask": tf.ones(
        (8, 512), dtype="int32"
    ),
}

input_dataset = tf.data.Dataset.from_tensor_slices(input_batch).batch(2)

for e in input_dataset:
    print(e["token_ids"].shape)
```

```
(2, 512)
(2, 512)
(2, 512)
(2, 512)
```